etc/disable-devel.inc

```diff
@@ -6,76 +6,53 @@ include /etc/firejail/disable-devel.local
 
 # GCC
 #blacklist /usr/lib/gcc	- seems to create problems on Gentoo
-blacklist /usr/bin/as
-blacklist /usr/bin/c++*
-blacklist /usr/bin/c8*
-blacklist /usr/bin/c9*
-blacklist /usr/bin/cpp*
-blacklist /usr/bin/g++*
-blacklist /usr/bin/gcc*
-blacklist /usr/bin/gdb
-blacklist /usr/bin/ld
-blacklist /usr/bin/x86_64-unknown-linux-gnu-gcc*
-blacklist /usr/bin/x86_64-linux-gnu-g++*
-blacklist /usr/bin/x86_64-linux-gnu-gcc*
-blacklist /usr/bin/x86_64-unknown-linux-gnu-g++*
+blacklist ${PATH}/as
+blacklist ${PATH}/c++*
+blacklist ${PATH}/c8*
+blacklist ${PATH}/c9*
+blacklist ${PATH}/cpp*
+blacklist ${PATH}/g++*
+blacklist ${PATH}/gcc*
+blacklist ${PATH}/gdb
+blacklist ${PATH}/ld
+blacklist ${PATH}/x86_64-unknown-linux-gnu-gcc*
```

Comment:
Are wildcards only possible at the end?
Otherwise we could add `*-gcc*` here for architectures other than x86_64.

Reply:
Good idea. I think wildcards work anywhere:
```
$ firejail --blacklist=/home/fred/*ocument*
$ cd Documents
bash: cd: Documents: Permission denied
$ ls Documents
ls: cannot open directory 'Documents': Permission denied
```
Would you blacklist `*-gnu-gcc*` or just `*-gcc*`

Reply:
I would use `*-gcc*`, because `-gnu-gcc` is for glibc-based compilers. musl for example uses the `x86_64-linux-musl` prefix instead.